src/main/kotlin/lotto/domain/WinningLotto.kt

```diff
@@ -0,0 +1,12 @@
+package lotto.domain
+
+data class WinningLotto(val lotto: Lotto, val bonusNumber: LottoNumber) {
```

Comment:
이 변수들은 외부에서 참조되는 코드가 없네요.
습관적으로 `private val`로 선언하고 꼭 필요한 경우에만 변경해주세요.
그리고 구현된 코드들을 보니 굳이 data class로 선언하지 않아도 잘 동작할 것 같아요.

```suggestion
class WinningLotto(
    private val lotto: Lotto,
    private val bonusNumber: LottoNumber
) {
```